### PR TITLE
Add docs for how to add Groups and their properties

### PR DESCRIPTION
**Pull request by bragur**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝Changed routes:


| [**…dd-a-group-to-an-event**](https://docs-khcej2ma8.avo-dev.app/data-design/how-to-add-a-group-to-an-event) |
|:-:|
|<a href="https://docs-khcej2ma8.avo-dev.app/data-design/how-to-add-a-group-to-an-event">Screenshot of /data-design/how-to-add-a-group-to-an-event</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/ZZCP5RAKK8jpnQJkSpYfFWPIp5HXhqABDDWO2lbk-SDkzxMtxnRhyv_weVfY-QCnUjIkJfpkew7bCTciZH5jeg.png">(view full size)</a> |







Commit c16924cdcc38fb64574aab644834da874cc0c99b (https://docs-khcej2ma8.avo-dev.app).

File: pages/data-design/how-to-add-a-group-to-an-event.mdx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+layout: 'guide'
+title: 'How to add a Group to an Event'
```
Comment (alex-tpom6oh) [inline pages/data-design/how-to-add-a-group-to-an-event.mdx:3]: Should we also describe the log event with group action here?

File: pages/data-design/how-to-add-a-group-to-an-event.mdx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+## Step 2 - Add Groups and Group Properties
+
+To link users with Groups click `+ Add Group Type` and choose the Group Type you wish to associate the user with. All subsequent calls will link the user to this Group.
+
+<img
+  src={require('../../images/data-design/add-group.png')}
+  alt="How to add a Group"
+/>
+
+To update Group Properties click `+ Add Group Properties` to add as many properties as you'd like from your groups. Avo will manage each separate call respectively. Note that Segment will always link a user with a Group when updating Group Properties.
```
Comment (alex-tpom6oh) [inline pages/data-design/how-to-add-a-group-to-an-event.mdx:26]: Can probably mention that Segment does not support log event with group, that's why Segment will always link a user with a Group when updating Group Properties.

**Comment by stefaniabje** [comment]
LGTM!

**Comment by stefaniabje** [comment]
I think we should merge this 💃

**Review by stefaniabje — APPROVED** [review]

**Review by alex-tpom6oh — APPROVED** [review]

**Pull request merged**